change/@azure-msal-common-2020-09-23-11-49-08-public-network-interface.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Remove null in function return types to be compatible with ICacheManager.ts",
```

Comment:
Can we update this as well and add the PR number at the end?

Reply:
Done